Dockerfile

```
@@ -1,38 +1,52 @@
-ARG VERSION=unspecified
+ARG GIT_COMMIT=unspecified
+ARG GIT_REMOTE=unspecified
+ARG VERSION=0.0.1
 
-FROM python:3.9.6-alpine
+FROM python:3.9.6
```

You should be able to get this to build using Alpine, which is our default standard for Docker projects.  You will need to use `apk` instead of `apt` to install packages.  If you run into any showstopper issues with Alpine, then we can discuss options for alternative environments.
```suggestion
FROM python:3.9.6-alpine
```

---

I have tested changing it locally to use alpine but am getting an error message when it tries to install the python package matplotlib.
Seems there are missing dependencies. Is installing all the extra dependencies advised?
https://stackoverflow.com/questions/65569248/how-to-install-matplotlib-on-alpine
Found this 
https://gist.github.com/orenitamar/f29fb15db3b0d13178c1c4dd611adce2

---

It's fine to install the dependencies needed to get matplotlib working on Alpine.

---

Hi @dav3r I have been trying several days to install matplotlib on Alpine. So far the linux dependencies are making the docker build time an additional 10 minutes and then still likely to get errors on other python packages supporting matplotlib.
If using Alpine could be made into an independent issue, that would be great as I see it is not a simply fix. Thanks.

---

That is fine- please create the issue and add a `TODO` comment in the code with the link to that issue.  Thanks!

---

Resolved in eb3bd0c69a710a7be49e2b3a1bae4fe71d9bc78f

---

Can you please correct the wording in the comment and the issue to reflect that the Dockerfile is currently using the [`python:3.9.6` image](https://hub.docker.com/layers/python/library/python/3.9.6/images/sha256-8a523686691e879ccea743a9baa327fa9ab51f273dc3a57ad02cf8f020c39b6b), not an Ubuntu image?

---

Resolved in 02a0524e945c56c0ec3c8709f9924fa56826aeb0

---

Please see https://github.com/cisagov/pca-report-generator-docker/pull/10#discussion_r750446697 for my suggested verbiage.

---

Resolved in 64810e12fd374547bf00eb4097b90362842bdb05

---

Thanks!